bundles/binding/org.openhab.binding.mios/src/main/java/org/openhab/binding/mios/internal/MiosUnitConnector.java

```diff
@@ -257,7 +258,29 @@ private void callMios(String url) {
 		logger.debug("callMios: Would like to fire off the URL '{}'", url);
 
 		try {
-			getAsyncHttpClient().prepareGet(url).execute();
+			@SuppressWarnings("unused")
+			Future<Integer> f = getAsyncHttpClient().prepareGet(url).execute(new AsyncCompletionHandler<Integer>() {
```

Comment:
does execute() provide a timeout parameter? It would be good to limit the timeout to something reasonable …